cl/search/models.py

```diff
@@ -81,7 +82,7 @@ class PRECEDENTIAL_STATUS:
 
 
 @pghistory.track(
-    pghistory.Snapshot(),
+    AfterUpdateOrDeleteSnapshot(),
```

Comment:
More triple-line issues here. I don't usually complain, but there are so many of them! :)